pom.xml

```
@@ -17,12 +17,12 @@
   <packaging>jar</packaging>
 
   <properties>
-    <java.version>17</java.version>
-    <edge-common-spring-version>2.4.6</edge-common-spring-version>
-    <folio-spring-cql-version>7.1.2</folio-spring-cql-version>
-    <openapi-generator.version>7.3.0</openapi-generator.version>
-    <spring-cloud-wiremock>4.0.2</spring-cloud-wiremock>
-    <swagger-parser.version>2.1.20</swagger-parser.version>
+    <java.version>21</java.version>
+    <edge-common-spring-version>3.0.0-SNAPSHOT</edge-common-spring-version>
+    <folio-spring-cql-version>8.2.2</folio-spring-cql-version>
```

```suggestion
    <folio-spring-cql-version>8.0.0</folio-spring-cql-version>
```
Sunflower version: https://github.com/folio-org/folio-spring-support/releases/tag/v9.0.0